scrapi/base/__init__.py

```diff
@@ -30,17 +31,25 @@ class BaseHarvester(object):
 
     @abc.abstractmethod
     def harvest(self, days_back=1):
-        pass
+        raise NotImplementedError
```

Comment:
I actually added these for coverage testing, so that it will ignore all lines that have ```raise NotImplementedError``` I suppose I could ignore pass instead, but I thought that had a higher chance of being real code.